app/controllers/posts_controller.rb

```diff
@@ -1,4 +1,5 @@
 class PostsController < ApplicationController
```

Comment:
- [ ] _LikesController_ is missing! The likes controller should have a _create_ action to enable 
 user to be able to add a like to posts, similar to the one shown below:

```ruby
  # controllers/likes_controller.rb

  class LikesController < ApplicationController
    ...

    def create
      @post = Post.find(params[:post_id])
      @like = @current_user.likes.new(post_id: @post.id)

      respond_to do |format|
        if @like.save
          format.html { redirect_to @post }
        else
          format.html { redirect_to @post }
        end
      end
    end
	  ...

  end
```